platform-operator/internal/k8s/certificate/certificate.go

```
@@ -79,6 +80,9 @@ func createTLSCert(log *zap.SugaredLogger, kubeClient kubernetes.Interface, comm
 	existingSecret, err := secretsClient.Get(context.TODO(), OperatorTLS, metav1.GetOptions{})
 	if err == nil {
 		log.Infof("Secret %s exists, using...", OperatorTLS)
+		log.Info("Adding labels in existing secret")
+		_ = updateSecretLabel(secretsClient, existingSecret)
```

Why do we ignore the error here?

---

I have handled the error now.